### PR TITLE
enable fractional gpu for aws

### PR DESCRIPTION
**Pull request by nikp1172**

### CODE REVIEW

File: charts/tfy-gpu-operator/templates/fractional-gpu-timeslicing.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{{- if .Values.clusterType.azureAks }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: time-slicing-config
```
Comment (shubhamrai1993) [inline charts/tfy-gpu-operator/templates/fractional-gpu-timeslicing.yaml:6]: Lets not hardcode but make it a variable and pick from values. Same for the number of replicas at line #15

File: charts/tfy-gpu-operator/templates/fractional-gpu-timeslicing.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{{- if .Values.clusterType.azureAks }}
```
Comment (chiragjn) [inline charts/tfy-gpu-operator/templates/fractional-gpu-timeslicing.yaml:1]: This seems incorrect?

File: charts/tfy-gpu-operator/values.yaml [hunk 3]
```diff
@@ -139,6 +142,10 @@ aws-eks-gpu-operator:
         - key: nvidia.com/gpu
           effect: NoSchedule
           operator: Exists
+        - key: nvidia.com/fractional-gpu
```
Comment (chiragjn) [inline charts/tfy-gpu-operator/values.yaml:145]: Maybe we should probably call this something like `nvidia.com/time-sliced-gpu` to be specific?

File: charts/tfy-gpu-operator/templates/fractional-gpu-timeslicing.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: time-slicing-config
+  namespace: {{ .Release.Namespace }}
+data:
+  fractional-10: |-
+    version: v1
```
Comment (chiragjn) [inline charts/tfy-gpu-operator/templates/fractional-gpu-timeslicing.yaml:10]: Does this apply globally or only to certain nodes?

**Pull request closed without merging**